docs/moving-an-app-off-static.md

```diff
@@ -0,0 +1,124 @@
+is included, and possibly in other controllers if they need to set slimmer_template.
+
+Make a note of controllers that set a different slimmer template. They may indicate a
+need either for a specialist layout, or for config variables to send to the default
+application layout.
+
+You may also need to remove slimmer references in test setups.
+
+## 2. Update the application layout to call the layout_for_public component directly
+
+[example commit](https://github.com/alphagov/email-alert-frontend/pull/1890/commits/f80d16bf9dd44093c5541b5802bafe2d41eea7bb)
+
+With static, app layouts are an HTML page, with a body that contains a wrapper element.
+This element is replaced by slimmer with the contents of the relevant template from
+static (which is in turn the body section of a layout_for_public component).
+
+Since layout_for_public is itself a full HTML page, you replace the entire section
+with a call to the layout_for_public component. You will need to add a `content_for :head`
+block. To this, add the gem component stylesheets for the components used in
+layout_for_public, and the call to `render_component_stylesheets`, and also
+any additional head elements that were in the existing layout.
+
+Since the app is now not getting the default styles from static, configure the
+publishing components gem to not exclude static css files.
+
+in `config/initializers/govuk_publishing_components.rb`:
+```
+GovukPublishingComponents.configure do |c|
+  c.component_guide_title = "Email Alert Frontend Component Guide"
+  c.application_stylesheet = "application"
+  c.exclude_css_from_static = false
+end
+```
+
+You will also need to make sure that the js dependencies are included in our application.js,
+
+```
+//= require govuk_publishing_components/dependencies
+//= require govuk_publishing_components/components/feedback
+//= require govuk_publishing_components/components/layout-super-navigation-header
+```
+
+...and that the application.scss contains at least the component support styles:
+
+```
+@import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/component_support";
+```
+
+...at this point, the app should be rendering correctly for the most part.
+
+## 3. Add specific layouts
+
+[example commit](https://github.com/alphagov/email-alert-frontend/pull/1890/commits/a8d28b5567dcd27442fda4c7ab20c3e2d6f502c3)
+
+If in step 1 you identified controllers that were using a slimmer template other than
+the default (`gem_layout`), you will now need to look at the relevant templates in static
+and decide whether the template used differs enough from the base template to warrant
+a separate layout. This is a judgement call - in the example above, the `gem_layout_account_manager`
+template requires a lot of configuration, so it seems simplest to keep it as a separate
+layout. If there's only one or two things different, possibly configuration is the way.
```

Comment:
Would it help to add some more detail to the end here? As a FE dev if I'd got to this point and decided that a separate layout isn't necessary, I'm not sure I'd know what to do next. Could there be an example of configuration?